bower.json

```diff
@@ -13,7 +13,15 @@
     "package.json"
   ],
   "dependencies": {
-    "d2l-colors": "^2.2.3",
-    "polymer": "^1.7.0"
+    "polymer": "Polymer/polymer#^1.9.1 || ^2.0.0",
+    "d2l-colors": "^2.2.3"
+  },
+  "variants": {
+    "1.x": {
+      "dependencies": {
+        "polymer": "Polymer/polymer#^1.7.0",
```

Comment:
Yeah I think we could go to `1.9.1` here, I think it's going to get installed anyway with `^1.7.1`?